### PR TITLE
Add the possibility to set column width

### PR DESCRIPTION
**Pull request by tobiaso88**
I think this would be a great feature to this package

### CODE REVIEW

**Comment by Gustavinho** [comment]
Excellent, great idea, just wondering if we can achieve the same using tailwindcss utilities instead of the `with` attribute

And another thing, could you update your PR with the latest changes of `release/2.1.0` branch, that branch is about to be released and it has the fixes about travis-ci

File: resources/views/components/table.blade.php [hunk 1]
```diff
@@ -14,7 +14,7 @@
     <tr>
       {{-- Renders all the headers --}}
       @foreach ($headers as $header)
-        <th class="px-3 py-3">
+        <th class="px-3 py-3" width="{{ is_object($header) ? $header->width : '' }}">
```
Comment (Gustavinho) [inline resources/views/components/table.blade.php:17]: Should we validate if `width` has a value?  or to set an empty string as a default value of this property? what do you think?

Comment (tobiaso88) [reply resources/views/components/table.blade.php]: I think that is a good idea

**Comment by tobiaso88** [comment]
We could use tailwindcss for this. But then we would be forced to use fixed widths.
I've rebased the lastest version.

**Comment by Gustavinho** [comment]
Thanks for updating your branch, there is a broken test btw, it seems a typo in the assertion
image

**Pull request merged**